### PR TITLE
Fix bugs in utils/aimsun/load.py

### PR DESCRIPTION
**Pull request by umangs94**
## Pull request information

- **Status**: ready to merge
- **Kind of changes**: bug fix

## Description

1. Updated the path to data.json
2. Added a check before querying centroid_config.origin_centroids and centroid_config.destination_centroids to prevent erroring out when the model doesn't have origin or destination centroids set.

### CODE REVIEW

File: flow/utils/aimsun/load.py [hunk 1]
```diff
@@ -132,7 +138,7 @@ def get_dict_from_objects(sections, nodes, turnings, cen_connections):
 model.load(template_path)
 
 # collect the simulation parameters
-params_file = 'flow/core/kernel/scenario/data.json'
+params_file = 'flow/core/kernel/network/data.json'
```
Comment (Yasharzf) [inline flow/utils/aimsun/load.py]: In lines 177 and 186 we have similar bugs. Could you also fix them (replace 'scenario' with 'network')?

Comment (umangs94) [reply flow/utils/aimsun/load.py]: In my local directory, those two files are still in `core/kernel/scenario`, not `network`. I don't think that they should be changed.

Comment (AboudyKreidieh) [reply flow/utils/aimsun/load.py]: yup, please do

Comment (umangs94) [reply flow/utils/aimsun/load.py]: Done!

Comment (AboudyKreidieh) [reply flow/utils/aimsun/load.py]: great, thanks!

**Pull request merged**